p2pserver/message/utils/msg_handler.go

```diff
@@ -441,16 +436,17 @@ func VerAckHandle(data *msgCommon.MsgPayload, p2p p2p.P2P, pid *evtActor.PID, ar
 				}
 				nodeConsensusAddr := addr[:i] + ":" +
 					strconv.Itoa(int(remotePeer.GetConsPort()))
-				go p2p.Connect(nodeConsensusAddr, true)
+				if remotePeer.GetConsPort() > 0 {
```

Comment:
change the upper if as:
       if config.DefConfig.P2PNode.DualPortSupport && remotePeer.GetConsPort() > 0

is 'return nil' ok "if i < 0"?
the following p2p.Send(msgpay.NewAddrReq()) is not required?

Reply:
refer to https://github.com/ontio/ontology/pull/178, the handler logic would remove error value.so all return error will delete in that pull.

NewAddrReq is needed for acquiring nbr node`s address.
other changes completed in latest code.